### ISSUE TITLE
External URL profile pictures

### ISSUE THREAD

**Issue by towerofnix** [decent-chat/decent#224]
There should be a way to set your profile picture to an external URL.

*However,* we have to consider security -- if somebody sets their profile picture to the URL of a server they're running, they could find out the IP addresses of people who view their profile picture (so, anybody in the chat room). Perhaps a client setting to load or not load external profile pictures? But in that case, it also makes sense to have a list of whitelisted domains, and Hmmmmmmmmmmmmmmmmmmmmmm feature creep???

Ping @eq.

**Comment by eq** [comment]
yeah it might be feature creep? i dunno the suggestion was a joke lol

**Comment by eq** [comment]
the first thing i'm going to do if this gets implemented is set the url to https://eq.ed1.club/spark.png

**Comment by towerofnix** [comment]
Alternatively, uploading profile pictures straight to Decent.

**Comment by eq** [comment]
Well, that's how a lot of services do it, with a fall back to libravatar at least. It makes sense, and I can still upload spark.png

**Comment by heyitsmeuralex** [comment]
Uploading direct sounds good. I don't like the idea of arbitrary URLs, though.
@eq how do we prevent against spark.png :package:
